underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,15 +124,21 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new SleepingRetry(MAX_TRY) {
```

Comment:
It's a little awkward to use `SleepingRetry` here since we don't actually sleep at all. Could we instead create a `CountRetry` implementing `RetryPolicy` which only allows `n` retries?